test/config_parser_SUITE.erl

```diff
@@ -2164,11 +2164,11 @@ test_mod_mam_meta(T, M) ->
     ?errf(T(#{<<"simple">> => <<"yes">>})),
     ?errf(T(#{<<"extra_fin_element">> => <<"bad_module">>})),
     ?errf(T(#{<<"extra_lookup_params">> => <<"bad_module">>})),
-    ?errf(T(#{<<"cache_config">> => #{<<"cache_module">> => <<"mod_wrong_cache">>}})),
```

Comment:
I think we could add a small test here with `module` set to `internal` and some options set to make sure it produces an error.